### ISSUE TITLE
Metadata might not show, depending on how it's stored

### ISSUE THREAD

**Issue by pegleGrot** [sumatrapdfreader/sumatrapdf#1846]
Document Properties (Ctrl-D) does not show metadata if it's stored in some unusual ways.
Some other viewers do show it, like PDF-XChange Viewer.

Two sample formats where Suamtra shows nothing:

[test_meta-dates.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/5743666/test_meta-dates.pdf)
[test_meta-producer_and_dates.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/5743667/test_meta-producer_and_dates.pdf)

**Comment by GitHubRulesOK** [comment]
I loaded both those files into a pdf meta editor (Normally used to remove XMP or other meta data)
Both are totally devoid of such entries so I would say Foxit is correct to say there is none
Here I saved the file 200 years ago but kept you as the author so as not to be called a  plagiarist
![image](https://user-images.githubusercontent.com/12754913/103160207-e0c5f380-47c9-11eb-922b-32e55930fd3d.png)

**Comment by pegleGrot** [comment]
Yes, it's not exactly standard metadata. Maybe according to the strict specs it should not be shown, but practically it's there (both files).

I think it's better for software to dig deeper and show anything there is to show. Both more informative, and better for sanitation checking.

**Comment by GitHubRulesOK** [comment]
Many a redactor has been caught out by feather duster cleaning the surface of a document, rather than tear down, discard and rebuild.
Many apps including SumatraPDF are limited to incremental update.
However others that rewrite and rebuild can still also carry over sensitive information.

The best solution is not to include any meta data at all. I believe the only secure method is for the Author to then wet sign their own signature and naturally the date should always be there on the scanned page. 

Then the digital file can be sealed with a lump of wax in a standalone bank vault server so no password cracker is able to access it.

**Comment by pegleGrot** [comment]
Not include any metadata - in many cases not possible, depending on the writing software and tools you use.

**Comment by GitHubRulesOK** [comment]
I have found incremental Meta Data very useful, especially for tracing infected MSOffice users, However Microsoft and others used to provide privacy tools to make that more difficult, one tool for example is from their sysinternals discussed elsewhere for removing Streams and you can also right click in explorer to remove some personal data.
Unfortunately ExifInfo is limited for editing many file types.

There are a couple of freeware tools but the one I used to test above was the discontinued https://web.archive.org/web/20170722230731/http://www.becyhome.de/becypdfmetaedit/description_eng.htm
You need to first load in repair mode (avoid incremental) and look at extras to check editions (if you wish to save version history etc.) then reload in complete rewrite and clear both the first two tabs which should do the trick of obliteration

**Comment by GitHubRulesOK** [comment]
@kjk the first file has no normal metadata and digging out the last copy of the suggested viewer for comparison with the first file It showed no more than SumatraPDF does apart from the 2 DATES which I would have expected from MuPDF However although they say modified they dont show either date
THUS the dates seem to **simply be the date of download and save TODAY** which is not really useful 

<img width="1292" height="710" alt="Image" src="https://github.com/user-attachments/assets/31ddec6a-48c1-4da5-9adb-07b250a878b2" />
<img width="433" height="80" alt="Image" src="https://github.com/user-attachments/assets/2c5a1c02-412d-4e57-8f93-b874b97647ba" />


The second example does have the producer and production date shown by old and newest tracker viewers as coming from the XMP and mupdf recognises there was extra modification! but they don't read the info in fact xmp is not even mentioned in their basic documentation :-)

<img width="287" height="120" alt="Image" src="https://github.com/user-attachments/assets/4cd36fe3-cfe8-4d2a-a949-75df1cd98007" />

<img width="481" height="179" alt="Image" src="https://github.com/user-attachments/assets/6a35baee-464a-4577-ab47-3e486cf3e9c5" />

<img width="231" height="52" alt="Image" src="https://github.com/user-attachments/assets/dc1432fe-3265-43e2-b272-f6e204e804ee" />

**Comment by pegleGrot** [comment]
What's that tool showing XMP Structure?

@GitHubRulesOK 
> the one I used to test above was the discontinued https://web.archive.org/web/20170722230731/http://www.becyhome.de/becypdfmetaedit/description_eng.htm You need to first load in repair mode (avoid incremental) and look at extras to check editions (if you wish to save version history etc.) then reload in complete rewrite and clear both the first two tabs which should do the trick of obliteration

The one I mentioned, where a complete rewrite still left some metadata, was probably BeCy PDFMetaEdit.
